### PR TITLE
Assignment2

### PR DESCRIPTION
**Pull request by qpengcheng**
tarea 2 corregido y tarea 3 realizado

### CODE REVIEW

File: Assignment2/qpengcheng-pengcheng.rdf [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	<rdf:Description rdf:about="#Computer101">
+	
+	<hasOwner rdf:resource = "#User10A"/>
+	
+	</rdf:Description>
+    
+	<rdf:Description rdf:about="#Measurement8401">
+	
+	<hasTemperature rdf:datatype="http://www.fi.upm.net/XMLSchema#integer"> 29 </hasTemperature>
+	<atTime rdf:datatype="http://www.fi.upm.net/XMLSchema#date"> 2010-06-12T12:00:12 </atTime>
```
Comment (ocorcho) [inline Assignment2/qpengcheng-pengcheng.rdf]: This should be dateTime

File: Assignment2/qpengcheng-pengcheng.rdf [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	
+	</rdf:Description>
+    
+	<rdf:Description rdf:about="#User10A">
+	
+	<hasName rdf:resource="#Pedro"/>
```
Comment (ocorcho) [inline Assignment2/qpengcheng-pengcheng.rdf]: This should be a string, not a resource.

File: Assignment2/qpengcheng-pengcheng.ttl [hunk 3]
```diff
@@ -0,0 +1,10 @@
+@base  <http://www.fi.upm.net/v1>
+@prefix vocab: <http://www.fi.upm.net/v1#>
+
+:Class01  		vocab:includes :Sensor029 ;
+				vocab:includes :Computer101.
+:Sensor029  	vocab:hasMeasurement :Measurement8401 ;
+:Computer101	vocab:hasOwner	:User10A;
+:Measurement8401 vocab:hasTemperature :"2";	#xsd:integer
```
Comment (ocorcho) [inline Assignment2/qpengcheng-pengcheng.ttl]: You should add datatypes

File: Assignment2/qpengcheng-pengcheng.ttl [hunk 4]
```diff
@@ -0,0 +1,12 @@
+@base  <http://www.fi.upm.net/v1>.
+@prefix vocab: <http://www.fi.upm.net/v1#>.
+@prefix hasTemperature :<http://www.fi.upm.net/v1#hasTemperature>
+@prefix atTime : <http://www.fi.upm.net/v1#atTime>
+@prefix hasName: <http://www.fi.upm.net/v1#hasName>
+:Class01  		vocab:includes :Sensor029 ;
+				vocab:includes :Computer101.
+:Sensor029  	vocab:hasMeasurement :Measurement8401 ;
+:Computer101	vocab:hasOwner	:User10A;
+:Measurement8401 hasTemperature:integer 2;
```
Comment (ocorcho) [inline Assignment2/qpengcheng-pengcheng.ttl]: This should be 29

Comment (ocorcho) [reply Assignment2/qpengcheng-pengcheng.ttl]: Furthermore, hasTemperature:integer does not make sense.

File: Assignment2/qpengcheng-pengcheng.ttl [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+				 atTime:dateTime 2010-06-12T12:00:12.
```
Comment (ocorcho) [inline Assignment2/qpengcheng-pengcheng.ttl]: You have to add a datatype. Check how others have done it.

Comment (qpengcheng) [reply Assignment2/qpengcheng-pengcheng.ttl]: done

**Comment by ocorcho** [comment]
Assignment 2 is correct. I will check later the assignment 3.

**Pull request merged**